src/main/java/org/broadinstitute/hellbender/tools/examples/metrics/multi/ExampleMultiMetrics.java

```diff
@@ -0,0 +1,50 @@
+package org.broadinstitute.hellbender.tools.examples.metrics.multi;
+
+import htsjdk.samtools.metrics.MetricBase;
+import org.broadinstitute.hellbender.metrics.MultiLevelMetrics;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.io.Serializable;
+
+/**
+ * An example multi-level metrics collector that just counts the number of reads (per unit/level)
+ */
+public class ExampleMultiMetrics extends MultiLevelMetrics implements Serializable {
+
+    private static final long serialVersionUID = 1;
+
+    // The final metric collected for each level and for all reads.
+    public int numReads;
```

Comment:
We might want to follow the unorthodox metrics naming scheme which makes them all UPPERCASE, along with a note explaining that this is the name that's used in the output file for the metrics.  It's very not java standards, but maybe consistency is better than following the standards here?